app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionViewModel.java

```diff
@@ -117,6 +117,8 @@ private int decideColSpan(@NonNull Session session, int roomCount) {
             return 3;
         } else if (session.isBreak()) {
             return roomCount;
+        } else if (session.isDinner()) {
+            return 6;
```

Comment:
Does `roomCount` always 6? If so, we could change `Type` to the following code.

```java
private enum Type {
        CEREMONY(3), SESSION(1), BREAK(6), DINNER(6);

        int colspan;

        Type(int colspan) {
            this.colspan = colspan;
        }

        @Override
        public String toString() {
            return super.toString().toLowerCase();
        }
    }
```

and then remove `isBreak()` etc methods, and these if/else statements.